playbooks/roles/jira/tasks/main.yml

```diff
@@ -0,0 +1,43 @@
+---
+# Jira role
+
+- name: install Jira dependencies
+  apt:
+    pkg:
+      - mysql-client
+
+- name: Ensure commands can be executed as www-data - set shell
+  user:
+    name: www-data
+    shell: /bin/false
+
+- name: Copy jira init script
+  copy:
+    src: jira.service
+    dest: /etc/systemd/system/jira.service
+  notify:
+    - restart jira
+
+- name: enable jira
+  systemd:
+    name: jira
+    enabled: yes
+    daemon_reload: yes
+
+- name: add backup pruner cron job
+  cron:
+    name: clean old Jira exports
+    state: present
+    job: find /var/www/jira/var/jira/export -maxdepth 1 -type f -mtime +150 -delete -regextype sed -regex '.*/[0-9]\{4\}-[A-Z][a-z]\{2\}-[0-9]\{2\}.*.zip'
```

Comment:
If you increase the maxdepth here, you can remove the corrupted backup pruner job

Reply:
@swalladge I don't really know if Jira can create more different backups/archives with the same name pattern in this directory so I prefer to have two commands. Does it sound reasonable?

Reply:
But the name pattern regex you use is the same in both.

Reply:
I don't really see the problem, but anyway, I have removed this line. 👍

Reply:
There isn't really any problem I guess; it's just a lot of duplication.  Either way. :)